tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/ClaimValueInjectionEndpoint.java

```diff
@@ -551,4 +554,28 @@ else if(Math.abs(customValue.doubleValue() - value.doubleValue()) < 0.00001) {
             .build();
         return result;
     }
+
+    @GET
+    @Path("/verifyInjectedCustomBoolean")
+    @Produces(MediaType.APPLICATION_JSON)
+    public JsonObject verifyInjectedCustomBoolean(@QueryParam("value") String value) {
+        boolean pass = false;
+        String msg;
+        // iat
```

Comment:
What does this comment refer to?

Reply:
Copy paste from previous method. Let me remove them.